src/c1_foeppl_von_karman/foeppl_von_karman_curvable_bell_elements.h

```diff
@@ -2134,9 +2134,9 @@ of freedom at internal points. They are {ux, uy}",
      {
       if (nod_pt->is_on_boundary(b))
        {
-#ifdef PARANOID
```

Comment:
I can tell you haven't tried compiling without PARANOID! I have moved the block start after nzeta is declared.